Makefile

```
@@ -50,16 +53,20 @@ rubygem/build/darwin-amd64/ejson: build/bin/darwin-amd64
 	mkdir -p $(@D)
 	cp -a "$<" "$@"
 
+rubygem/build/freebsd-amd64/ejson: build/bin/freebsd-amd64
+	mkdir -p $(@D)
+	cp -a "$<" "$@"
+
 rubygem/build/linux-amd64/ejson: build/bin/linux-amd64
 	mkdir -p $(@D)
 	cp -a "$<" "$@"
 
 cmd/$(NAME)/version.go: VERSION
-	echo 'package main\n\nconst VERSION string = "$(VERSION)"' > $@
+	printf 'package main\n\nconst VERSION string = "$(VERSION)"' > $@
```

Isn't this missing a newline?
```suggestion
	printf 'package main\n\nconst VERSION string = "$(VERSION)"\n' > $@
```